File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/widgetassertions/TreeViews_ViewMyParentStringRowHandlesRow.cpp

#include "TreeViews_ViewMyParentStringRowHandlesRow.h"
#include "TreeViews_ViewMyParentStringRowHandlesRow.h"
#include <string>

namespace widgetassertions
{
  std::string TreeViews_ViewMyParentStringRowHandlesRow::getValuesLabelText( ) 
  {
    return this->ValuesLabelText;
  }
  std::string TreeViews_ViewMyParentStringRowHandlesRow::getRowHandle( ) 
  {
    return this->RowHandle;
  }
  std::string TreeViews_ViewMyParentStringRowHandlesRow::getParentRowHandle( ) 
  {
    return this->ParentRowHandle;
  }
}

File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/widgetassertions/TreeViews_ViewMyParentStringRowHandlesRow.cs
/// <filename>
///     TreeViews_ViewMyParentStringRowHandlesRow.cs
/// </filename>


namespace widgetassertions
{
    internal class TreeViews_ViewMyParentStringRowHandlesRow
  {
        private string ValuesLabelText;
     
        public     virtual string getValuesLabelText()
    {
      return this.ValuesLabelText;
    }
     
        private string RowHandle;
     
        public     virtual string getRowHandle()
    {
      return this.RowHandle;
    }
     
        private string ParentRowHandle;
     
        public     virtual string getParentRowHandle()
    {
      return this.ParentRowHandle;
    }
  }
}

